tests/visualization_tests/test_timeline.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from io import BytesIO
+from typing import List
+
+import pytest
+
+import optuna
+from optuna.study.study import Study
+from optuna.trial import TrialState
+from optuna.visualization._plotly_imports import go
+from optuna.visualization._timeline import _get_timeline_info
+from optuna.visualization._timeline import plot_timeline
+
+
+def _create_study(trial_states_list: List[TrialState]) -> Study:
+    study = optuna.create_study()
+    fmax = float(len(trial_states_list))
+    for i, s in enumerate(trial_states_list):
+        study.add_trial(
+            optuna.trial.create_trial(
+                params={"x": float(i)},
+                distributions={"x": optuna.distributions.FloatDistribution(-1.0, fmax)},
+                value=0.0,
+                state=s,
+            )
+        )
+    return study
+
+
+def test_get_timeline_info_empty() -> None:
+    study = _create_study([])
```

Comment:
We can directly call `optuna.create_study` here to simplify the logic.
```suggestion
    study = optuna.create_study()
```